packages/public/server/src/module/Subject/config/instances.config.php

```diff
@@ -360,7 +362,21 @@
                         'applet',
                     ],
                 ],
-                'test area' => [
```

Comment:
Did you intentionally remove the test area of spanish and hindi tenant?

Reply:
yes, spanish and hindi tenant dont have a taxonomy yet so this is not set up yet and it was removed in https://en.serlo.org/test-area and https://de.serlo.org/testbereich, so I figured we don't need this in new tenants.